client/src/views/Groups.tsx

```diff
@@ -43,70 +46,91 @@ export const Groups = () => {
         return groups.filter(group => group.deletedDateTime)
     }, [groups])
 
-    const columnDefGroups: ColDef[] = [{ field: 'displayName' }, { field: 'description' }, {
+    const columnDefGroups: ColDef[] = [{ field: 'displayName', headerName: 'Display Name', minWidth: 160, maxWidth: 240, flex: 2 }, { field: 'description', minWidth: 220, maxWidth: 580, flex: 2 , headerName: 'Description' }, {
```

Comment:
having miniWidith and maxWidth makes table less scalable, please avoid as much as possible

Reply:
This is removed now, i also agree it makes table less scalable, main reason i added was to accomodate long description which is ... if it overflows, now i added flex, but for that i need to fix width of first column, rest of other columns use flex values which are approximated.